app/src/pages/inside/launchesPage/modals/moveToDebugModal/moveToDebugModal.jsx

```diff
@@ -55,7 +121,10 @@ export class MoveToDebugModal extends Component {
         okButton={okButton}
         cancelButton={cancelButton}
       >
-        <p className={cx('message')}>{Parser(intl.formatMessage(messages.moveToDebugText))}</p>
+        <p className={cx('message')}>
+          {(data.ids.length > 1 && Parser(intl.formatMessage(messages.moveToDebugMultipleText))) ||
```

Comment:
I think ternary operator will be more readable here

Reply:
Fixed